spec/mailers/subscription_mailer_spec.rb

```diff
@@ -39,7 +40,36 @@
         body = SubscriptionMailer.deliveries.last.body.encoded
         expect(body).to include "This order was automatically created for you."
         expect(body).to_not include "Unfortunately, not all products that you requested were available."
-        expect(body).to include "href=\"#{spree.order_url(order)}\""
+      end
+    end
+
+    describe "linking to order page" do
+      let(:order_link_href) { "href=\"#{spree.order_url(order)}\"" }
+      let(:order_link_style) { "style='[^']+'" }
+
+      let(:shop) { create(:enterprise, allow_order_changes: true) }
+
+      let(:email) { SubscriptionMailer.deliveries.last }
+      let(:body) { email.body.encoded }
+
+      before do
+        SubscriptionMailer.placement_email(order, {}).deliver
+      end
+
+      let(:customer) { create(:customer, enterprise: shop) }
```

Comment:
is there a reason to have this let after the before and not together with the others lets above?